src/main/webapp/app/communication/post/post.component.ts

```diff
@@ -85,6 +85,7 @@ export class PostComponent extends PostingDirective<Post> implements OnInit, OnC
     // if the post is previewed in the create/edit modal,
     // we need to pass the ref in order to close it when navigating to the previewed post via post title
     modalRef = input<NgbModalRef | undefined>(undefined);
+    searchQuery = input<string | undefined>(undefined);
```

Comment:
Would it work here to use `input<string>('')` instead here, avoiding the undefined? Or would you need to adapt the `courseWideSearchConfig().searchTerm` then?